### DISCUSSION TITLE
Brackets 2.1.2 Release

### DISCUSSION THREAD

**Discussion by abose** [brackets-cont/brackets#208] [Announcements]
**_Brackets 2.1.2 Release is here!_**
* Download Release Windows installer: [Brackets 2.1.2.exe]() 
  - Installer Built and Signed by [phcode.dev](https://phcode.dev/) Project.
* Mac installer: [brackets.2.1.2.signed.dmg]()

### Known Issues
   * Layout issues with brackets-git extension. The extension needs to be updated to use the new navbar.

### All new File navigation bar with icons for
   * locate currently working file in file tree(binocular icon)
   * Navuigate back and forward icon
   * split view icon
   * find in files icon

![Brackets_Jq8l5AlHFY](https://user-images.githubusercontent.com/5336369/167259952-c6475770-6e3e-4b7e-bc12-1a13ddd92c7f.gif)
### Live preview enhancement
* Drag any number in code with `ctrl+alt` or `cmd+alt` pressed to see instant changes
* Select any number in code with `ctrl+alt` + `up/down` arrow keys
Credits- https://github.com/peterflynn

![chrome_1NMucPeeju](https://user-images.githubusercontent.com/5336369/167288309-eee1a944-80a3-4da7-b8b8-eeaab4ec1f6e.gif)
### Github and Gitlab URL open Integration 
Press `ctrl-shift-o` in windows or `cmd-shift-o` and paste a github file url to open in brackets. Eg. open this URL in brackets https://github.com/brackets-cont/brackets/blob/master/package.json

![Brackets_J9JhfJLT75](https://user-images.githubusercontent.com/5336369/167257521-17f316c3-c2e8-4652-8efb-55f37224b07c.gif)

<hr /><em>This discussion was created from the release <a href='https://github.com/brackets-cont/brackets/releases/tag/v2.1.2-release'>Brackets 2.1.2 Release</a>.</em>

**Comment by p-linnane** [comment]
Is this a pre-release or a full release? I've seen it tagged as both.

**Reply by abose** [reply]
This is a full release. the pre-release version is 2.1.1.

**Reply by p-linnane** [reply]
Got it. The release page states:
>Brackets 2.1.2 Pre-Release is here!

**Reply by charlypa** [reply]
@p-linnane 
Typo updated

**Comment by sasha-by** [comment]
When vertical aligne of window name will be fixed in macos? 
<img width="1496" alt="image" src="https://user-images.githubusercontent.com/79199287/180385904-2f00444d-05b1-48e5-bdc3-430916168511.png">

**Reply by wojtaszczyk** [reply]
Still same problem..
<img width="1278" alt="Zrzut ekranu 2023-06-14 o 21 49 36" src="https://github.com/brackets-cont/brackets/assets/136643613/b0c4ce19-a132-435c-aba4-4eb8c39b35a5">

**Comment by Herzogdb** [comment]
Is there going to be a version that runs on Mac Chips M1 and M2 natively. When opening v2.1.2, I get an error when "To open 'Microsoft AU Daemon' you need to install Rosetta. If I hit NOT NOW, nothing happens.

**Reply by abose** [reply]
Native builds are scheduled for development but it is a few months out. In the meantime, Checkout phcode.dev too though its still in pre alpha build.

**Comment by PunyTRexArms** [comment]
**Thanks to the developers for all their hard work in making Brackets v2 possible.** I'm wondering what the current time table looks like for a Linux release. Version 1.14.1 was working more or less okay until the upgrade to Ubuntu 22.04 LTS. At this point it is largely broken, which was inevitable.

**Comment by flaviovivorio** [comment]
Hi. Is there any Linux version on the way? That would be so amazing!
